refs/w3c.json

```diff
@@ -36597,6 +36600,21 @@
             "hr-time"
         ]
     },
+    "hr-time-20120313": {
+        "aliasOf": "hr-time-1-20120313"
+    },
+    "hr-time-20120522": {
+        "aliasOf": "hr-time-1-20120522"
+    },
+    "hr-time-20121023": {
+        "aliasOf": "hr-time-1-20121023"
+    },
+    "hr-time-20121217": {
+        "aliasOf": "hr-time-1-20121217"
+    },
+    "hr-time-20191121": {
+        "aliasOf": "hr-time-1-20191121"
+    },
```

Comment:
Is there anyway this could be baked into the `switchWith` logic? It feels like this would solve a lot of our other related problems.

Reply:
Done in&nbsp;https://github.com/tobie/specref/pull/572/commits/3054155463a9e3d6d50e48db7ac484dec3b5dd6e.